RecoCTPPS/PixelLocal/interface/CTPPSPixelLocalTrackProducer.h

```diff
@@ -0,0 +1,75 @@
+/*
+ *
+* This is a part of CTPPS offline software.
+* Author:
+*   Fabrizio Ferro (ferro@ge.infn.it)
+*   Enrico Robutti (robutti@ge.infn.it)
+*   Fabio Ravera   (fabio.ravera@cern.ch)
+*
+*/
+#ifndef RecoCTPPS_PixelLocal_CTPPSPixelLocalTrackProducer_H
+#define RecoCTPPS_PixelLocal_CTPPSPixelLocalTrackProducer_H
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "DataFormats/Common/interface/DetSetVector.h"
+#include "DataFormats/Common/interface/DetSet.h"
+#include "FWCore/Framework/interface/stream/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "DataFormats/Common/interface/Handle.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+ 
+#include "FWCore/Framework/interface/ESWatcher.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/Utilities/interface/InputTag.h"
+
+#include "Geometry/Records/interface/VeryForwardRealGeometryRecord.h"
+
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelRecHit.h"
+#include "DataFormats/CTPPSReco/interface/CTPPSPixelLocalTrack.h"
+#include "DataFormats/DetId/interface/DetId.h"
+
+
+#include "RecoCTPPS/PixelLocal/interface/RPixDetPatternFinder.h"
+#include "RecoCTPPS/PixelLocal/interface/RPixDetTrackFinder.h"
+
+#include <string>
+#include <vector>
+
+class CTPPSPixelLocalTrackProducer : public edm::stream::EDProducer<>
```

Comment:
header file for producers is not necessary, the declaration and implementation can all be moved to CTPPSPixelLocalTrackProducer.cc